### ISSUE TITLE
[USERSNAP] Proposal type visuals need improvement

### ISSUE THREAD

**Issue by dmtrjsg** [Joystream/pioneer#2949]
**Taken at**: https://dao.joystream.org/#/proposals/past
**Fields**:
- Houston, we have a problem.: Proposal type visuals need improvement
- Comment: Position, size and color all does not work.

We are clearly seeing the title is serving he primary purpose of conveying the information, so the type information needs to be more clearly secondary in terms of attention. We still need it, because people can unwhittingly or maliciously set a misleading title.

A simple fix is to place the title a subtext under the main title, and change background color to something drawing less attention, or even consider dropping it entirely. Probably requires making each row taller also, which is fine.
- Feedback: bedeho

![SCREENSHOT](https://resources.usersnap.com/company/f9f719df-a929-4461-bb6e-46dbfbceffb7/datapoint_screenshot/c00db144-98a1-4535-9a65-fed782da95e4-annotated_19434bb7-9497-4a1a-a333-9ee3badaf618.png?etag=ef86b868a501a05dd61f29ffdbf0e3ae)
*[Download original image](https://resources.usersnap.com/company/f9f719df-a929-4461-bb6e-46dbfbceffb7/datapoint_screenshot/c00db144-98a1-4535-9a65-fed782da95e4-annotated_19434bb7-9497-4a1a-a333-9ee3badaf618.png?etag=ef86b868a501a05dd61f29ffdbf0e3ae)*

**Browser**: Firefox 99 (macOS Catalina)
**Screen size**: 5120x1440
**Browser size**: 1478x920
**[Open #232 in Usersnap](https://app.usersnap.com/l/feedback/aa8f9a8a-a034-4c7a-9dab-f46d3b7ada2c?utm_medium=integration&utm_campaign=integration&utm_content=open)**

Powered by **[Usersnap](https://usersnap.com/?utm_source=product&amp;utm_medium=poweredbylink&amp;utm_campaign=github_entry)**.

**Comment by traumschule** [comment]
agreed. also [patterns](https://uxdesign.cc/viewing-patterns-the-subconscious-psychology-of-the-eye-8b8b8522f753) like subtle persistent type based colors will help trained members to parse the list faster, for example
- colored bubble with proposal title as text and proposal type as tooltip
- clicking type bubble filters proposals by this type
- clicking status filters proposals by this status (similar to labels)
- clicking member filters by creator etc.
- rationale is usually most important info => show below title (saves extra clicks to open preview and expand rationale)
- merge `current` & `past` tabs to avoid another extra click (use [Switch](https://retool.com/components/switch) with all proposals already on state for instant update)
- filters row: drop-downs are slow, user has to expand, parse, scroll, click, wait. better for example
  - [floating action buttons](https://material.io/archive/guidelines/components/buttons-floating-action-button.html), one per filter type with clear icons (calendar for time, head for proposer, checkmark for status, ...)
  - that expand on hover (or click on mobile): `useState`-based filters via [buttons](https://www.freecodecamp.org/news/how-to-make-a-filter-component-in-react/)/[segmented controls](https://medium.com/@kollinz/dropdown-alternatives-for-better-mobile-forms-53e40d641b53)/[checkbox group](https://retool.com/components/checkbox-group)
  - w/o requiring extra QN request (having latest 100 proposals on state should not take too much memory / time to fetch on start)
  - mid-term: user study to determine most accepted option (by builders?)

There are probably more modern patterns but these could be implemented within a day.

*I know the filter part is pretty packed, do UI designers understand what i mean?*

**Comment by traumschule** [comment]
Wide tables can be a problem on smaller screens. Some fields could be hidden via CSS.

Accounts on tiled screen
![acc-sm-scrn](https://user-images.githubusercontent.com/31551045/209950093-6cdd04e4-adb6-42aa-a3ad-5df54d42f7b2.png)
Only at 75% zoom all columns become visible.

Related: #1794 (moved from #3997)
